scalacheck-effect/src/test/scala/org/specs2/scalacheck/effect/ScalaCheckEffectMatchersApiSpec.scala

```diff
@@ -0,0 +1,105 @@
+   Arbitrary
+   to specify a specific arbitrary instance for a parameter
+   ${propF { (i: Int) => IO(i must be_>(0)) }.setArbitrary(positiveInts)}
+   to specify a specific arbitrary instance for any parameter
+   ${propF { (s: String, j: Int) => IO(j must be_>(0)) }.setArbitrary2(positiveInts)}
+   to specify all the arbitrary instances
+   ${propF { (i: Int, j: Int) => IO(i + j must be_>(0)) }.setArbitraries(positiveInts, positiveInts)}
+
+   Gen
+   to specify a specific generator for a parameter
+   ${propF { (i: Int) => IO(i must be_>(0)) }.setGen(positiveInts.arbitrary)}
+   to specify a specific generator for any parameter
+   ${propF { (s: String, j: Int) => IO(j must be_>(0)) }.setGen2(positiveInts.arbitrary)}
+   to specify all the generators
+   ${propF { (i: Int, j: Int) => IO(i + j must be_>(0)) }.setGens(positiveInts.arbitrary, positiveInts.arbitrary)}
+
+   Shrink
+   to specify a specific shrink instance for a parameter
+   ${propF { (i: Int) => IO(i === i) }.setShrink(shrinkInts)}
+   to specify a specific shrink instance for any parameter
+   ${propF { (s: String, j: Int) => IO(true) }.setShrink2(shrinkInts)}
+   to specify all the shrink instances
+   ${propF { (i: Int, j: Int) => IO(i === i) }.setShrinks(shrinkInts, shrinkInts)}
+
+   Pretty
+   to specify a specific pretty instance for a parameter
+   ${propF { (i: Int) => IO(i === i) }.setPretty(prettyInts)}
+   ${propF { (i: Int) => IO(i === i) }.pretty(_.toString)}
+   to specify a specific pretty instance for any parameter
+   ${propF { (s: String, j: Int) => IO(true) }.setPretty2(prettyInts)}
+   ${propF { (s: String, j: Int) => IO(true) }.pretty2(_.toString)}
+   to specify all the pretty instances
+   ${propF { (i: String, j: Int) => IO(i === i) }.setPretties(prettyStrings, prettyInts)}
+   ${propF { (i: String, j: Int) => IO(i === i) }.pretties(_.toString, _.toString)}
+   to specify the pretty for collected data
+   ${propF { (i: String, j: Int) => IO(i === i) }.collectAll
+    .prettyFreqMap((fq: FreqMap[Set[Any]]) => fq.total.toString)}
+   ${propF { (i: String, j: Int) => IO(i === i) }.collectAll.prettyFreqMap(_.toString)}
+
+   Collect
+   to specify a specific collect function for a parameter
+   ${propF { (i: Int) => IO(i === i) }.collect}
+   ${propF { (i: Int) => IO(i === i) }.collectArg((i: Int) => math.abs(i))}
+   to specify a specific collect function for any parameter
+   ${propF { (s: String, j: Int) => IO(true) }.collect2}
+   ${propF { (s: String, j: Int) => IO(true) }.collectArg2(math.abs)}
+   to specify all the collect functions
+   ${propF { (i: Int, j: Int) => IO(i === i) }.collectAll}
+   ${propF { (i: Int, j: Int) => IO(i === i) }.collectAllArgs(math.abs, math.abs)}
+
+ Test parameters can also be specified
+   the minimum number of ok tests
+   minTestsOk1
+
+   the verbosity of a property can be turned on and off
+   verbose1
+
+  """
+  // def minTestsOk1: Prop =
```

Comment:
Why are those 2 examples commented out? Any problem with them?

Reply:
They use `propBoolean` which isn't working with ScalaCheck Effect. Or at least, I wasn't sure how to make it work.

Reply:
I see, and the `==>` function also does not exist. Since those are dummy assertions anyway, feel free to replace them with `1 === 1` or something equivalent.